### ISSUE TITLE
Set initial tab

### ISSUE THREAD

**Issue by ChiliConSql** [joshforisha/ember-foundation#7]
I just started using this add-on. Great work so far! Thank you.
Is there a way to set the first tab to 'active' when initializing? If not, what do you think is the best way to accomplish this, so that data is displayed and the tab is highlighted?

I put in this hack, and it seems to work, what do you think?

    // addon/components/f-tab-panel.js
    // line 11
    title: this.getAttribute('data-tab-title'),
    class: this.getAttribute('class')

then in my template:

    {{#f-tab-pane id='details' title='Details' class='active'}}

**Comment by joshforisha** [comment]
That sort of works, but is probably not the best pattern long-term.

I went ahead and added a new option for the f-tab-panel, `activeTabId` to pass in the string name of the tab you want to be active initially. I bumped the addon version to 0.6.0 also. Let me know if you have any issues with it; I still need to add tests :/

I'll likely update the addon sometime this week to Ember CLI 0.2.7 (or newer if it releases before then), so I might revisit some of the components' properties then.
